.env.example

```diff
@@ -13,6 +13,6 @@ DB_ENTITIES=dist/src/modules/**/entities/**/*.entity{.ts,.js}
 DB_MIGRATIONS=dist/**/migrations/*{.ts,.js}
 DB_TYPE=postgres
 DB_SSL=true
-
+CLIENT_URL=
```

Comment:
whats this for, a variable with no valu, please clarify or remove ?

Reply:
It's used in the waitlist email sent to the user for accessing the website for more information.